Extension/src/main.ts

```diff
@@ -38,7 +38,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<CppToo
 
     let errMsg: string = "";
     const arch: string = PlatformInformation.GetArchitecture();
-    if (arch !== 'x64' && (process.platform !== 'win32' || (arch !== 'x86' && arch !== 'arm64')) && (process.platform !== 'linux' || (arch !== 'x64' && arch !== 'arm' && arch !== 'arm64')) && (process.platform !== 'darwin' || arch !== 'arm64')) {
+    if (arch !== 'x64' && (process.platform !== 'win32' || (arch !== 'x86' && arch !== 'arm64')) && ((process.platform === 'win32' || process.platform === 'darwin') || (arch !== 'arm' && arch !== 'arm64')) && (process.platform !== 'darwin' || arch !== 'arm64')) {
```

Comment:
I find this expression too complicated to parse.  Is there a way to make it more human-readable?

Reply:
Yeah, but I didn't bother, i.e. pre-existing issue.